src/main/java/dev/snowdrop/release/services/JiraClientConfiguration.java

```diff
@@ -20,6 +20,8 @@
 
 import com.atlassian.jira.rest.client.api.JiraRestClient;
 import com.atlassian.jira.rest.client.internal.async.AsynchronousJiraRestClientFactory;
+import io.quarkus.arc.DefaultBean;
+import io.quarkus.arc.properties.IfBuildProperty;
```

Comment:
This import seems unnecessary.

Reply:
`IfBuildProperty` is not used but `DefaultBean` is.